<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addProduct_master</class>
 <widget class="QDialog" name="Dialog_addProduct_master">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>219</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>219</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>219</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_add_product">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_product_name">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Товар</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_product_cost">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>109</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Добавление товара&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
